src/components/Payroll/PayrollOverview/SubmissionBlockers/GenericBlocker.tsx

```diff
@@ -0,0 +1,45 @@
+import type { PayrollSubmissionBlockersType } from '@gusto/embedded-api/models/components/payrollsubmissionblockerstype'
+import { Flex } from '@/components/Common'
+import { useComponentContext } from '@/contexts/ComponentAdapter/useComponentContext'
+import { useDateFormatter } from '@/hooks/useDateFormatter'
+
+interface GenericBlockerProps {
+  blocker: PayrollSubmissionBlockersType
+  selectedValue?: string
+  onUnblockOptionChange: (blockerType: string, value: string) => void
+}
+
+export const GenericBlocker = ({
```

Comment:
Have we tested this with other blockers in practice? i'm wondering if it's best to just add generic language here around contacting support instead of rendering the radio unless we have an inventory of other submission blockers and have actually tested this with those

Reply:
Good call out, I made it generic right now and added a KNOWN BLOCKERS WITH RESOLUTIONS const. So for now we only can resolve fast_ach issues. Everything else will display a generic error for now encouraging them to contact support.